evennia/contrib/wilderness.py

```diff
@@ -0,0 +1,490 @@
+"""
+Wilderness system
+
+Evennia contrib - titeuf87 2017
+
+This contrib provides a wilderness map. This is an area that can be huge where
+the rooms are mostly similar, except for some small cosmetic changes like the
+room name.
+
+Usage:
+
+    This contrib does not provide any commands. Instead the @py command can be
+    used.
+
+    A wilderness map needs to created first. There can be different maps, all
+    with their own name. If no name is provided, then a default one is used.
+
+    @py from evennia.contrib import wilderness; wilderness.create_wilderness()
+
+    Once created, it is possible to move into that wilderness map:
+    @py from evennia.contrib import wilderness; wilderness.enter_wilderness(me)
+
+
+Customisation:
+
+    The defaults, while useable, are meant to be customised. When creating a new
+    wilderness map it is possible to give a "map provider": this is a python
+    object that is smart enough to create the map.
```

Comment:
You should give a brief example of a Map Provider here, especially which methods the wilderness system expects it to have.